cmd/telegraf/telegraf.go

```diff
@@ -277,7 +280,14 @@ func runAgent(ctx context.Context,
 
 	logger.SetupLogging(logConfig)
 
-	log.Printf("I! Starting Telegraf %s", version)
+	log.Printf("I! Starting Telegraf %s%s", version, internal.Customized)
+	log.Printf("I! Available plugins: %d inputs, %d aggregators, %d processors, %d parsers, %d outputs",
```

Comment:
I love this additional output, thank you!

I built with a config of just `inputs.diskio` and `outputs.file` and I ended up seeing:

```
2022-08-16T17:43:50Z I! Available plugins: 3 inputs, 0 aggregators, 0 processors, 0 parsers, 2 outputs
```

Which are:

```
telegraf-sven on  minify [?] via 🐹 v1.19 
❯ ./telegraf --input-list
Available Input Plugins:
  diskio
  io
  system
telegraf-sven on  minify [?] via 🐹 v1.19 
❯ ./telegraf --output-list
Available Output Plugins: 
  file
  wavefront
```

Two questions:

- Why is wavefront pulled in? I understand the other inputs due to dependencies
- Should we document this in the README, additional plugins may get pulled in due to dependencies. This could be a single sentence added to the readme.

Reply:
It seems `wavefront` is coming through the backyard. ;-) The route is

- `file` output imports `github.com/influxdata/telegraf/plugins/serializers`
- `serializers/registry.go` imports `github.com/influxdata/telegraf/plugins/serializers/wavefront`
- `wavefront` serializer imports `github.com/influxdata/telegraf/plugins/outputs/wavefront` :fearful: 
 
This happens with the epic comment:
> // TODO: this dependency is going the wrong way: Move MetricPoint into the serializer.

:man_facepalming: 

I guess this has to be fixed, but is not an issue of `custom_builder`... What do you think?

Reply:
> Should we document this in the README, additional plugins may get pulled in due to dependencies. This could be a single sentence added to the readme.

Absolutely. Will do.

Reply:
lol @ that comment, let's circle around and fix that after this is merged. I can file an issue.